app/assets/stylesheets/provider/admin/cms/_sidebar.scss

```
@@ -18,7 +19,14 @@
   }
 
   & + #tab-content {
+    background-color: $white;
+    padding: line-height-times(1);
```

```suggestion
    padding: line-height-times(1) 0 1.5rem 1.5rem
```

as now you are overwriting the the rest of the padding which is correct